README.md

```diff
@@ -107,13 +107,15 @@ spec.source_files = 'Argo/**/*.{h,swift}', 'Carthage/Checkouts/Runes/Source/Rune
 
 ## What's included? ##
 
-Importing Runes introduces 3 new operators and one global function:
+Importing Runes introduces several new operators and one global function:
 
 - `<^>` (pronounced "map")
 - `<*>` (pronounced "apply")
 - `>>-` (pronounced "flatMap") (left associative)
 - `-<<` (pronounced "flatMap") (right associative)
 - `pure` (pronounced "pure")
+- `>->` (pronounced "Kleisli arrow") (left associative)
```

Comment:
This is technically correct, but maybe we could call it "Monadic compose" instead? Seems a _little_ more approachable in my opinion.